gui/webdisplay/src/RWebWindowsManager.cxx

```diff
@@ -154,6 +155,41 @@ void RWebWindowsManager::AssignWindowThreadId(RWebWindow &win)
    }
 }
 
+//////////////////////////////////////////////////////////////////////////////////////////
+/// If LC_ROOT_LISTENER_SOCKET variable is configured,
+/// message will be send to that unix socket
+
+bool RWebWindowsManager::InformListener(const std::string &msg)
+{
+#ifdef R__WIN32
+   (void) msg;
+   return false;
+
+#else
+
+   const char *fname = gSystem->Getenv("ROOT_LISTENER_SOCKET");
+   if (!fname || !*fname)
+      return false;
+
+   TSocket s(fname);
+   if (!s.IsValid())
+      return false;
```

Comment:
Should this be emitting a warning / error message?

Reply:
`ROOT_LISTENER_SOCKET` is nice addon, but not really important - just new window may not be handled.
But I can add warning that socket is gone or not accessible.